common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -93,17 +93,48 @@
     <%doc>
         Loads a studio-frontend page, with the necessary context. Context is expected
         as a dictionary in the body of this tag.
-
-        Dev note: we could also add the locale-injection script in this block
-            -use a better default than the hardcoded 'en'. There should be a setting or something?
-            -lookup (webpack exported) locale-injection script using lang as key
-            -include it as the first script in this block
     </%doc>
     <%
         body = capture(caller.body)
+        body_dict = json.loads(body)
+        locale = body_dict['lang']
+        messages = "{}"
+        localeData = ""
+
+        if locale != 'en':
+            # because en is the default, studio-frontend will have it loaded by default
+            ietf_to_transifex = {
+                'fr': 'fr_CA',
+                'zh-CN': 'zh_CN',
+                'es-419': 'es_419',
+                'ar': 'ar'
+            }
+
+            messages_path = "{base}/studio-frontend/dist/i18n/messages/{locale}.json".format(
+                base=settings.STATIC_ROOT_BASE,
+                locale=ietf_to_transifex[locale]
+            )
```

Comment:
@thallada, this is the part I was having some trouble with. The files are also copied into `common/vendor/something`, should I prefer that path and hard-code it?